### PR TITLE
Add recipe for rpdb

### PR DESCRIPTION
**Pull request by Bezier89**

### CODE REVIEW

File: recipes/rpdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - rpdb
+
+about:
+  home: http://tamentis.com/projects/rpdb
+  license: ISC (ISCL)
+  license_family: OTHER
+  license_file:
```
Comment (scopatz) [inline recipes/rpdb/meta.yaml]: Please include the license file, see https://conda-forge.org/docs/meta.html#packaging-the-license-manually if needed

File: recipes/rpdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - rpdb
+
+about:
+  home: http://tamentis.com/projects/rpdb
+  license: ISC (ISCL)
+  license_family: OTHER
+  license_file: 
+  summary: pdb wrapper with remote access via tcp socket
+  doc_url: 
+  dev_url:
```
Comment (scopatz) [inline recipes/rpdb/meta.yaml]: Please remove these two lines if they are empty

**Comment by scopatz** [comment]
Thanks for putting this in @Bezier89!

File: recipes/rpdb/meta.yaml (hunk 2 above)
Comment (Bezier89) [reply recipes/rpdb/meta.yaml]: Done (added dev_url, removed doc_url)

File: recipes/rpdb/meta.yaml (hunk 1 above)
Comment (Bezier89) [reply recipes/rpdb/meta.yaml]: Done (used license file from https://github.com/tamentis/rpdb/blob/master/LICENSE.txt)

**Comment by Bezier89** [comment]
Thanks for the review @scopatz. I believe I've addressed your comments.

File: recipes/rpdb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - rpdb
+
+about:
+  home: http://tamentis.com/projects/rpdb
+  license: ISC (ISCL)
+  license_family: OTHER
+  license_file: LICENSE.txt
```
Comment (scopatz) [inline recipes/rpdb/meta.yaml]: This needs to have the proper value to pick up the license file, see https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Comment (Bezier89) [reply recipes/rpdb/meta.yaml]: I built this package locally and it worked fine. I've also verified that it does in fact include the LICENSE.txt file (and if you change the recipe to look for e.g. `LICENSE.foo` it fails as expected).

This is using conda-build version 3.17.6. If the long form is needed for historical reasons or compatibility with older versions, let me know and I'll make the change.

Comment (scopatz) [reply recipes/rpdb/meta.yaml]: We may in the futrure have to drop back to older versions of conda-build. Please feel free to open a PR to the docs to open up a discussion on the format of this line

**Comment by scopatz** [comment]
Thanks for putting this in @Bezier89!

File: recipes/rpdb/meta.yaml (hunk 3 above)
Comment (Bezier89) [reply recipes/rpdb/meta.yaml]: Went ahead and made the change. Seperately, opened https://github.com/conda-forge/conda-forge.github.io/pull/693 to discuss updating documentation.

Comment (scopatz) [reply recipes/rpdb/meta.yaml]: Thanks!

**Pull request merged**